packages/ide-extension/src/telemetry/action-map.ts

```diff
@@ -94,7 +102,27 @@ export const actionMap: {
     }),
     [RESET_FILTERS]: (): TelemetryUIClearFiltersProps => ({
         action: 'CLEAR_FILTERS'
-    })
+    }),
+    [GET_BOOKMARKS]: (action: SendTelemetry): TelemetryUILoadBookmarksProps => ({
+        action: 'LOAD_BOOKMARKS',
+        count: Object.keys((action.payload.action as GetBookmarks).payload).length.toString()
+    }),
+    [UPDATE_BOOKMARKS]: (
+        action: SendTelemetry
+    ): TelemetryUIAddBookmarksProps | TelemetryUIRemoveBookmarksProps | TelemetryUISyncBookmarksProps => {
+        const bookmarkKey = (action.payload.action as UpdateBookmarks).payload.bookmarkKey;
+
+        if (bookmarkKey === '') {
+            return { action: 'SYNC_BOOKMARKS' };
```

Comment:
What do you think about removing this and make an own mapping react action `SYNCHRONIZE_BOOKMARK` -> telemetry action `BOOKMARK_CLICKED`?